### ISSUE TITLE
issue for problematic images

### ISSUE THREAD

**Issue by cbeauhilton** [vanderbilt-internal-medicine/vimbook#11]
this issue is a placeholder for messed up/missing images.

Add each as a new comment, will delete when dealt with (or leave behind comments if they will be useful for posterity).

**Comment by chasejwebber** [comment]
I updated the .index page to 6th edition however somehow it corrupted the image of the QR code

**Comment by chasejwebber** [comment]
hospital-medicine-telemetry.md  - need new image of table, currently as a corrupted image code from a couple years ago

**Comment by cbeauhilton** [comment]
> I updated the .index page to 6th edition however somehow it corrupted the image of the QR code
> 
> never mind, it was just showing up weird in the preview but committed just fine! disregard!

yeah, there are some slight differences between "GitHub-Flavored Markdown" and what mkdocs uses, so occasionally those kinds of hiccups occur.
